packages/components/src/draggable/index.js

```diff
@@ -9,6 +9,29 @@ const cloneHeightTransformationBreakpoint = 700;
 const clonePadding = 0;
 const bodyClass = 'is-dragging-components-draggable';
 
+/**
+ * @typedef RenderProp
+ * @property {(event: import('react').DragEvent) => void} onDraggableStart `onDragStart` handler.
+ * @property {(event: import('react').DragEvent) => void} onDraggableEnd `onDragEnd` handler.
+ */
+
+/**
+ * @typedef Props
+ * @property {(props: RenderProp) => JSX.Element | null} children Children.
+ * @property {(event: import('react').DragEvent) => void} [onDragStart] Callback when dragging starts.
+ * @property {(event: import('react').DragEvent) => void} [onDragOver] Callback when dragging happens over ???.
```

Comment:
Nit: Did we mean to put something in the place of `???`?

Reply:
Oh yes, I wasn't sure how best to describe this prop 🤔 Any good ideas?

Reply:
"The draggable element" perhaps?